docs/self-managed/operational-guides/backup-restore/backup-and-restore.md

```diff
@@ -64,7 +64,12 @@ If any of the steps above fail, you may have to restart with a new backup id. En
 
 To restore a Camunda 8 cluster from a backup, all components must be restored from their backup corresponding to the same backup id:
 
-1. Restore the state of [Operate](/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md), [Tasklist](/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md), and [Optimize](/self-managed/operational-guides/backup-restore/optimize-backup.md).
-2. Restore `zeebe-records*` indices from Elasticsearch snapshot.
-3. Restore [Zeebe](/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md).
-4. Start Zeebe, Operate, Tasklist, and Optimize.
+1. Start Zeebe, Operate, Tasklist, and Optimize. (To ensure templates/aliases etc. are created)
+2. Confirm proper configuration (such as shards, replicas count, etc.)
+3. Stop Operate, Tasklist, and Optimize.
+4. Delete all indices.
+5. Restore the state of [Operate](/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md), [Tasklist](/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md), and [Optimize](/self-managed/operational-guides/backup-restore/optimize-backup.md).
+6. Restore `zeebe-records*` indices from Elasticsearch snapshot.
+7. Restore [Zeebe](/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md).
+8. Start Zeebe, Operate, Tasklist, and Optimize.
+9. Wait until all components are restored to their respective backup ids (see [how to monitor a Camunda 8 cluster](./monitoring.md)).
```

Comment:
Where is `monitoring.md`? I couldn't find a page or a section that looked like it may fit here.

Reply:
Good catch, I see now this was only in the docs part for next version, and I suspect it was an old link which I accidently copied in. Removing it to keep it consistent.